vm/systemSmartContracts/delegation.go

```diff
@@ -1968,6 +2042,48 @@ func (d *delegation) saveGlobalFundData(globalFundData *GlobalFundData) error {
 	return nil
 }
 
+func (d *delegation) checkOwnerInitialFunds(delegationConfig *DelegationConfig, caller []byte, callValue *big.Int) error {
+	// initial owner funds must be 0 or higher than min deposit
+	if delegationConfig.InitialOwnerFunds.Cmp(zero) > 0 {
```

Comment:
Can initialOwnerFunds be slightly higher than 0 but less than the minDeposit value? Can we have this situation?

Reply:
no. only 0 or more than initialOwnerFunds

Reply:
ok